tests/basic/volume_ops_test.go

```diff
@@ -3652,4 +3653,81 @@ var _ = Describe("{OverCommitVolumeTest}", func() {
 		})
 
 	})
-})
\ No newline at end of file
+})
+var _ = Describe("{RestartPxandRestartNode}", func() {
+	/*
+	   https://purestorage.atlassian.net/browse/PTX-24483
+	   1.Deploy Applications
+	   2.Validate Applications are Deployed
+	   3.Restart Portworx service on few nodes
+	   4.once portworx is up on the node,immediately reboot the node
+	   5.Make sure Both portworx and node are up.
+	   6.Validate the Applications are running
+	*/
+	var contexts []*scheduler.Context
+	JustBeforeEach(func() {
+		StartTorpedoTest("RestartPxandRestartNode",
+			"Restart Portworx and Restart Node", nil, 0)
+	})
+	itLog := "RestartPxandRestartNode"
+	It(itLog, func() {
+		log.InfoD(itLog)
+		pxNodes := node.GetWorkerNodes()
```

Comment:
use GetStorageDriverNodes as this list will include torpedo node also incase of cloud platforms